larpandoracontent/LArObjects/LArTrackTwoViewOverlapResult.h

```diff
@@ -199,6 +229,36 @@ inline unsigned int TwoViewTransverseOverlapResult::GetNSamplingPoints() const
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+inline unsigned int TwoViewTransverseOverlapResult::GetNMatchedSamplingPoints() const
+{
+    if (m_isInitialized)
+        return m_nMatchedSamplingPoints;
+   
+    throw pandora::StatusCodeException(pandora::STATUS_CODE_NOT_INITIALIZED);
+}
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+
+inline unsigned int TwoViewTransverseOverlapResult::GetNReUpsampledSamplingPoints() const
+{
+    if (m_isInitialized)
+        return static_cast<unsigned int>(m_downsamplingFactor*m_nSamplingPoints);
+   
+    throw pandora::StatusCodeException(pandora::STATUS_CODE_NOT_INITIALIZED);
+}
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+
+inline unsigned int TwoViewTransverseOverlapResult::GetNMatchedReUpsampledSamplingPoints() const
+{
+    if (m_isInitialized)
+        return static_cast<unsigned int>(m_downsamplingFactor*m_nMatchedSamplingPoints);
```

Comment:
static cast the downsampling factor, rather than the entire thing.

Reply:
Actually don't do that, but static cast `m_nMatchedSamplingPoints` to a float and then keep the unsigned int static cast for the entire thing.